doc/timegapsplit.ipynb

```diff
@@ -2,14 +2,53 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
```

Comment:
Could you remove the pip install here?

Reply:
Also, feel free to add documentation for your setting, and usecase, here. You don't have to do it, but if typically people won't notice the feature unless it is in a getting started guide.

Reply:
yes ofc, just an artifact of me failing to properly use virtual environments